### PR TITLE
Update vietnamese translation

### PR DESCRIPTION
**Pull request by quangtrongonline**
Update vietnamese translation

### CODE REVIEW

**Comment by LukeTowers** [comment]
This pull request has been made to the wrong branch. Please review [the contributing guidelines](https://github.com/octobercms/october/blob/master/CONTRIBUTING.md#pull-requests) as all PRs need to be made to the develop branch.

I'll fix it for you this time, but please ensure you make any future PRs to the develop branch, not the master branch.

File: modules/backend/lang/vn/lang.php [hunk 1]
```diff
@@ -0,0 +1,588 @@
+<?php
+
+return [
+    'auth' => [
+        'title' => 'Administration Area'
```
Comment (LukeTowers) [inline modules/backend/lang/vn/lang.php]: Please don't leave untranslated strings in here.

**Comment by quangtrongonline** [comment]
hello, I will pay attention to the branch. I've updated some string in that file

File: modules/cms/lang/vn/lang.php [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+        'author_placeholder' => 'Tên cá nhân hoặc công ty',
+        'description_label' => 'Mô tả',
+        'description_placeholder' => 'Mô tả theme',
+        'homepage_label' => 'Trang chủ',
+        'homepage_placeholder' => 'Đường dẫn trang chủ tác giả',
+        'code_label' => 'Code',
+        'code_placeholder' => 'Tạo mã code duy nhất để phân phối theme của bạn',
+        'preview_image_label' => 'Hình ảnh xem trước theme',
+        'preview_image_placeholder' => 'Đường dẫn hình ảnh xem trước của theme.',
+        'dir_name_label' => 'Directory name',
```
Comment (LukeTowers) [inline modules/cms/lang/vn/lang.php]: There's still a bunch of untranslated strings in here, please remove them. The reason I don't want them in translation files is that the language system already falls back to the default translation (English), so it's redundant to have them repeated and also doesn't propagate the changes to the English translation by default.

Comment (quangtrongonline) [reply modules/cms/lang/vn/lang.php]: I've updated some string in that file

File: modules/system/lang/vn/validation.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
+<?php
+
+return [
+
```
Comment (LukeTowers) [inline modules/system/lang/vn/validation.php]: Don't include this file if it doesn't have any translations please

Comment (quangtrongonline) [reply modules/system/lang/vn/validation.php]: I have compiled it

File: modules/system/lang/vn/validation.php [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    |
+    | Here you may specify custom validation messages for attributes using the
+    | convention "attribute.rule" to name the lines. This makes it quick to
+    | specify a specific custom language line for a given attribute rule.
+    |
+    */
+
+    'custom' => [
+        'attribute-name' => [
+            'Tên_key' => 'giá_trị',
```
Comment (LukeTowers) [inline modules/system/lang/vn/validation.php]: Please don't add project specific custom attributes in the default language files; this is something that should be handled at a project level by making the changes in lang/system/vn/validation.php

File: modules/system/lang/vn/lang.php [hunk 5]
```diff
@@ -0,0 +1,434 @@
+<?php
+
+return [
+    'app' => [
+        'name' => 'QuangtrongOnlineCMS',
```
Comment (LukeTowers) [inline modules/system/lang/vn/lang.php]: Please don't add project specific custom translations in the default language files; this is something that should be handled at a project level by making the changes in lang/system/vn/lang.php.

**Comment by LukeTowers** [comment]
@quangtrongonline thanks for all your work on this so far, I'll keep reviewing it so that when it's merged it'll be top quality! Let me know if you have any questions.

**Comment by quangtrongonline** [comment]
yes, I hope my contributions will be helpful for your application.

File: modules/system/lang/vn/lang.php [hunk 6]
```diff
@@ -0,0 +1,434 @@
...
+        'pt-br' => 'Português (Brasil)',
+        'pt-pt' => 'Português (Portugal)',
+        'ro' => 'Română',
+        'ru' => 'Русский',
+        'fi' => 'Suomi',
+        'sv' => 'Svenska',
+        'sk' => 'Slovenský',
+        'tr' => 'Türkçe',
+        'uk' => 'Українська мова',
+        'vn' => 'Tiếng việt',
```
Comment (LukeTowers) [inline modules/system/lang/vn/lang.php]: You need to include this `'vn' => 'Tiếng việt',` in the system/lang/en/lang.php as well as it's the fallback for any missing translations in every other language file

File: modules/system/lang/vn/validation.php (hunk 4 above)
Comment (LukeTowers) [reply modules/system/lang/vn/validation.php]: Still need to remove this

File: modules/system/lang/vn/lang.php (hunk 6 above)
Comment (quangtrongonline) [reply modules/system/lang/vn/lang.php]: ok, i've update

Comment (quangtrongonline) [reply modules/system/lang/vn/lang.php]: but i dont find the "locale" in some language

File: modules/system/lang/vn/validation.php (hunk 4 above)
Comment (LukeTowers) [reply modules/system/lang/vn/validation.php]: Still need to remove this :)

Comment (quangtrongonline) [reply modules/system/lang/vn/validation.php]: done :)

**Comment by quangtrongonline** [comment]
I have updated the file. Tell me what's wrong

File: modules/system/lang/vn/client.php [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    |--------------------------------------------------------------------------
+    | Client-side Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | These are messages made available to the client browser via JavaScript.
+    | To compile this file run: php artisan october:util compile lang
+    |
+    */
+
+    'markdowneditor' => [
```
Comment (LukeTowers) [inline modules/system/lang/vn/client.php]: These translations are still missing

File: modules/system/lang/vn/lang.php [hunk 8]
```diff
@@ -0,0 +1,434 @@
...
+    'file' => [
+        'create_fail' => 'Không thể tạo file: :name'
+    ],
+    'combiner' => [
+        'not_found' => "Không tìm thấy combiner file ':name'."
+    ],
+    'system' => [
+        'name' => 'System',
+        'menu_label' => 'System',
+        'categories' => [
```
Comment (LukeTowers) [inline modules/system/lang/vn/lang.php]: These values aren't translated

File: modules/system/lang/vn/lang.php [hunk 9]
```diff
@@ -0,0 +1,434 @@
...
+        'sender_name' => 'Tên người gửi',
+        'sender_email' => 'Email người gửi',
+        'php_mail' => 'PHP mail',
+        'smtp' => 'SMTP',
+        'smtp_address' => 'Địa chỉ SMTP',
+        'smtp_authorization' => 'Yêu cầu cấp quyền',
+        'smtp_authorization_comment' => 'Check box này nếu SMTP sever của bạn yêu cầu cấp quyền truy cập(requires authorization).',
+        'smtp_username' => 'Tên đăng nhập',
+        'smtp_password' => 'Mật khẩu',
+        'smtp_port' => 'SMTP port',
```
Comment (LukeTowers) [inline modules/system/lang/vn/lang.php]: From here down isn't translated

File: modules/system/lang/vn/lang.php [hunk 10]
```diff
@@ -0,0 +1,434 @@
...
+        'deleting' => 'Đang xóa mẫu...',
+        'deleting_layout' => 'Đang xóa Layout...',
+        'sending' => 'Đang gửi mail test...',
+        'return' => 'Quay lại trang danh sách mail mẫu'
+    ],
+    'mail_brand' => [
+        'menu_label' => 'Mail branding',
+        'menu_description' => 'Chỉnh sửa màu sắc, giao diện của mẫu mail.',
+        'page_title' => 'Tùy chỉnh giao diện của mẫu mail',
+        'sample_template' => [
```
Comment (LukeTowers) [inline modules/system/lang/vn/lang.php]: From here down isn't translated

**Comment by LukeTowers** [comment]
@quangtrongonline reviewed it again, sorry for the delay, had surgery yesterday.

File: modules/system/lang/vn/lang.php (hunk 8 above)
Comment (quangtrongonline) [reply modules/system/lang/vn/lang.php]: i've update it

**Comment by quangtrongonline** [comment]
i've update, please review again :)

**Comment by quangtrongonline** [comment]
I think some terms should not be translated

**Comment by quangtrongonline** [comment]
hey, have you see my pull requests

**Comment by LukeTowers** [comment]
@quangtrongonline is it finished and ready for review now?

**Comment by quangtrongonline** [comment]
yes, i've update it. let me know if have problem

**Comment by LukeTowers** [comment]
Excellent job @quangtrongonline; thank you very much for your contribution!

**Pull request merged**